db/migrate/20160924010603_create_users.rb

```diff
@@ -6,6 +6,7 @@ def change
       t.string :last_name
       t.string :email
       t.string :password_digest
+      t.string :status, default: "pending"
```

Comment:
@kmbradshaw don't forget to remove this. are you going to add the new column migration or did I say I would do that?